src/main/java/baseball/domain/Result.java

```diff
@@ -0,0 +1,51 @@
+package baseball.domain;
+
+import baseball.domain.constants.NumberBaseballConstants;
+
+public class Result {
+
+    private final int ball;
+    private final int strike;
+    private final int numberLength;
+
+    private Result(int ball, int strike, int numberLength) {
+        this.ball = ball;
+        this.strike = strike;
+        this.numberLength = numberLength;
+    }
+
+    public static Result create(Answer answer, Answer guessAnswer) {
+        if(answer.getSize() != guessAnswer.getSize()) throw new IllegalArgumentException("Result 인스턴스를 만들 수 없습니다.");
+
+        int ballCount = 0;
+        int strikeCount = 0;
+
+        for(int i=0; i<guessAnswer.getAnswer().size(); i++) {
+            int number = guessAnswer.getAnswer().get(i);
+            int index = answer.getAnswer().indexOf(number);
+
+            if(index == -1) continue;
+
+            if(index == i) strikeCount++;
+            else ballCount++;
```

Comment:
자바 컨벤션을 보면 조건문을 한 줄에 이어서 사용하는 것보다 중괄호를 사용하는 것을 권장합니다!
```suggestion
            if(index == -1) {
               continue;
             }

            if(index == i) {
              strikeCount++;
            }
            else {
              ballCount++;
            }
```

![image](https://github.com/woowacourse-precourse/java-baseball-6/assets/73726591/36786752-96c0-40e5-b0d4-58ac6c050235)

Reply:
감사합니다!